### DISCUSSION TITLE
Issues with land cover classification

### DISCUSSION THREAD

**Discussion by bweeding** [UMEP-dev/UMEP#287] [Q&A] [answered]
Hi all, I'm trying to use the land cover reclassifier on the attached .tif file:
![image](https://user-images.githubusercontent.com/57697604/121633111-8f829580-cac5-11eb-8204-0b0432ab9789.png)

However, when I attempt to use the classifier, I receive the following error:
![image](https://user-images.githubusercontent.com/57697604/121633276-d2dd0400-cac5-11eb-8294-cdc0644fefb9.png)
![image](https://user-images.githubusercontent.com/57697604/121633337-f2742c80-cac5-11eb-96b5-88badb7eda6f.png)

Any suggestions as to what I'm doing wrong would be greatly appreciated!

[landcover_raw.zip](https://github.com/UMEP-dev/UMEP/files/6635937/landcover_raw.zip)

**Comment by biglimp** [comment]
You need to fill in numbers in the "greater than" column also. Your screen dump show that this column is empty.

**Reply by bweeding** [reply]
I have now done this and it is generating the landcover file. However, when I attempt to run the model using the file, I receive the error:

QgsProcessingException: Error in land cover grid: Land cover grid includes Confier land cover class. Ground cover information (underneath canopy) is required.

How do I go about generating the ground cover information under the canopy?

Cheers
Ben

**Reply by biglimp** [reply]
Well, when deriving land cover from lidar data this is a decission you need to do without knowing what is actually underneath. I usually reclass tress to bare soil which is the most common lc under a tree. However, is some cases there could be grass or paved. You simply have to make a judgement.

**Comment by bweeding** [comment, marked as answer]
Ah ok cool. How do I tell the model what I have decided? With a separate
TIF file?

**Reply by biglimp** [reply]
Just reclassify your conifer and deciduous to e.g. Bare soil

**Comment by bweeding** [comment]
Ah so the trees are only represented in the cdsm, not in the land cover?

**Reply by biglimp** [reply]
Correct. Land cover represent what material is on the ground, underneath the trees.

**Reply by bweeding** [reply]
Ah excellent, I've got it working now! Thank you.
